full-backend-tests/src/test/java/org/graylog/testing/utils/SearchUtils.java

```diff
@@ -33,37 +33,37 @@ public final class SearchUtils {
     private SearchUtils() {
     }
 
-    public static List<String> searchForAllMessages(RequestSpecification requestSpecification) {
+    public static List<String> searchForAllMessages(GraylogApis api) {
```

Comment:
What about passing a `Supplier<RequestSpec>` here, instead of the complete `GraylogApis`, preventing coupling to it?